server/app/views/admin/AdminLayout.java

```diff
@@ -164,7 +168,12 @@ private NavTag renderNavBar() {
     }
 
     return adminHeader.with(
-        headerLink("Logout", logoutLink, "float-right").withId("logout-button"));
+        headerLink("Logout", logoutLink, "float-right").withId("logout-button"),
+        featureFlags.getFlagEnabledFromConfig(ADMIN_SETTINGS_PANEL_ENABLED).orElse(false)
+            ? a(Icons.svg(Icons.COG).withClasses("h-6 w-6"))
+                .withHref(settingsLink)
+                .withClasses("float-right")
```

Comment:
non-blocking: Set the opacity to .75 to match the reset of the non-hover links. If the svg can handle changing the opacity on hover you can set that to 1. This way the cog will fit in better.

Reply:
Done